File: src/classes/recipe.ts
export class Recipe {
    id: number;
    recipeName: string;
    categoryId:number;
    preparationTime:number;
    levelDifficulty:number;
    dateAdded:Date;
    listComponent:string[];
    preparationMethod:string[];
    userId:number;
    picture:string;
 
  constructor(){}
    // constructor(id: number, 
    //     name: string ,
    //     categoryId:number,
    //     preparationTime:number,
    //     LevelDifficulty:number,
    //     dateAdded:Date,
    //     listComponent:string[],
    //     preparationMethod:string[],
    //     userId:number,
    //     picture:string,) {
    //   this.id = id;
    //   this.name = name;
    //   this.categoryId=categoryId;
    //   this.preparationTime=preparationTime;
    //   this.LevelDifficulty=LevelDifficulty;
    //   this.dateAdded=dateAdded;
    //   this.listComponent=listComponent;
    //   this.preparationMethod=preparationMethod;
    //   this.userId=userId;
    //   this.picture=picture;
    // }
  }

File: server/server/Recipe.cs
namespace server
{
    public class Recipe
    {
        public int Id { get; set; }
        public string? RecipeName { get; set; }
        public int CategoryId { get; set; }
        public double? preparationTime { get; set; }
        public int? LevelDifficulty { get; set; }
        public DateTime? DateAdded { get; set; }
        public List<string>? ListComponent { get; set; }
        public List<string>? preparationMethod { get; set; }
        public int? UserId { get; set; }
        public string? Picture { get; set; }



    }
}
